spec/configuration_spec.rb

```diff
@@ -140,6 +140,15 @@
         end
       end
     end
+  end
+
+  describe 'modifying default bank from memory' do
+    it 'creates new banks when needed' do
+      old_bank = Money.default_bank
+      bank = Money::RatesStore::Memory.new
 
+      Money.with_bank(bank) {}
```

Comment:
You definitely wanna make sure that the block was called and the new bank is set correctly within it